server/jetty/src/main/java/io/deephaven/server/jetty/JettyConfig.java

```diff
@@ -134,9 +145,31 @@ public static Builder buildFromConfig(Configuration config) {
         if (maxHeaderRequestSize != null) {
             builder.maxHeaderRequestSize(Integer.parseInt(maxHeaderRequestSize));
         }
+        builder.allowedHttpMethods(allowedHttpMethods);
+        builder.extraHeaders(extraHeaders);
         return builder;
     }
 
+    private static Map<String, String> readExtraHeaders(Configuration config) {
```

Comment:
Is it worth sharing any utilities between 11 and 12?

Reply:
I'd vote no, just so we don't have to keep track back and forth? Definitely could be persuaded here, but the third server implementation (netty) can't/shouldn't serve this.